.github/workflows/release.yml

```diff
@@ -27,8 +27,9 @@ jobs:
 
       - name: Release new version
         run:
-          mvn -B release:clean release:prepare release:perform -P release
+          mvn -Dmaven.test.skip=true -ntp -B release:clean release:prepare release:perform -P release
         env:
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
-          MAVEN_GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
\ No newline at end of file
+          MAVEN_GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@ckipp01 
do you know what am I missing?
the job fails with
```
Error:  Failed to execute goal org.apache.maven.plugins:maven-release-plugin:3.0.1:prepare (default-cli) on project scoverage-maven-plugin: Unable to commit files
Error:  Provider message:
Error:  The git-push command failed.
Error:  Command output:
Error:  git@github.com: Permission denied (publickey).
Error:  fatal: Could not read from remote repository.
```
i've flollowed your example in https://github.com/scalacenter/bloop-maven-plugin/blob/main/.github/workflows/release.yml

Reply:
Hey, I have no idea sorry.